### PR TITLE
upxtreme: Initial commit

### PR DESCRIPTION
**Pull request by LeanSheng**
Signed-off-by: Lean Sheng Tan <tanleansheng@outlook.com>

### CODE REVIEW

File: src/mainboard/aaeon/upsquared/fixed-dtfs.dts [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
     flash-info {
         compatible = "ore-flashinfo";
-        board-name = "UP Xtreme";
+        board-name = "UP Sqared";
```
Comment (rjoleary) [inline src/mainboard/aaeon/upsquared/fixed-dtfs.dts:9]:
```suggestion
        board-name = "UP Squared";
```

File: src/mainboard/aaeon/upsquared/fixed-dtfs.dts [hunk 2]
```diff
@@ -6,9 +6,9 @@
 
     flash-info {
```
Comment (rjoleary) [inline src/mainboard/aaeon/upsquared/fixed-dtfs.dts:7]: I think you should delete this file -- it's upsquared while everything else is upxtreme.

**Comment by rjoleary** [comment]
The CI system is failing with this format error:

```
Diff in /home/vsts/work/1/s/src/mainboard/aaeon/upxtreme/src/main.rs at line 14:
 
 use rpp_procedural::preprocess_asm;
 
-use fsp_common as fsp;
 use fsp_cfl_sys as fsp64;
+use fsp_common as fsp;
```

File: src/mainboard/aaeon/upsquared/fixed-dtfs.dts (hunk 1 above)
Comment (LeanSheng) [reply src/mainboard/aaeon/upsquared/fixed-dtfs.dts:9]: i fix this so it wont have duplicate name with the upxtreme board. Is this fine with you?

**Review by rminnich — APPROVED** [review]

**Pull request merged**